providers/shared/inputseeders/shared/id.ftl

```diff
@@ -0,0 +1,133 @@
+[#ftl]
+
+[@addInputSeeder
+    id=SHARED_INPUT_SEEDER
+    description="Shared inputs"
+/]
+
+[@addSeederToInputStage
+    inputStage=COMMANDLINEOPTIONS_SHARED_INPUT_STAGE
+    inputSeeder=SHARED_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputStage=MASTERDATA_SHARED_INPUT_STAGE
+    inputSeeder=SHARED_INPUT_SEEDER
+/]
+
+[@addSeederToInputStage
+    inputStage=QUALIFY_SHARED_INPUT_STAGE
+    inputSeeder=SHARED_INPUT_SEEDER
+/]
+
+[#macro shared_input_loader path]
+    [#assign shared_cmdb_masterdata =
+        (
+            getPluginTree(
+                path,
+                {
+                    "AddStartingWildcard" : false,
+                    "AddEndingWildcard" : false,
+                    "MinDepth" : 1,
+                    "MaxDepth" : 1,
+                    "FilenameGlob" : "masterdata.json"
+                }
+            )[0].ContentsAsJSON
+        )!{}
+    ]
+[/#macro]
+
+[#function shared_input_masterdata_seeder filter state]
```

Comment:
Our convention for dynamic functions is probably better as 

`<provider>_<function>_<qualifers>` 

Since the seeder is a functionality wouldn't it be better as 

`shared_inputseeder_masterdata` 
`sahred_inputloader` 

etc

Reply:
I've put examples on the azure provider 
https://github.com/hamlet-io/engine-plugin-azure/pull/224#pullrequestreview-594903506

Reply:
OK - I wasn't aware we were that strict but I can align to that, noting that "provider" is a seeder. For instance, with the next set of changes re command line options, an entrance can provide a seeder.

Reply:
Updated.